web/src/engine/element-wrappers/src/designIFrame.ts

```diff
@@ -31,7 +31,7 @@ class StyleCommand {
   }
 }
 
-export default class DesignIFrame extends OutputTarget {
+export default class DesignIFrame extends OutputTarget<[]> {
```

Comment:
I don't understand why this differs from line 23 of contentEditable.pas above? Why are we extending Array here but Object there?

Reply:
Good catch.  Those were meant to be curly (Object) here too.